lib/nodejs/swagger/__tests__/swagger.test.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+
+describe('swagger.Swagger', () => {
+
+  // const mockValidate = () => jest.fn()
+  //   .mockImplementation(() => Promise.resolve('VALIDATED_SCHEMA'));
+  const mockValidate = () => Promise.resolve({schema: 'VALIDATED_DEREFERENCED_SCHEMA'});
+  const mockBundle = () => Promise.resolve({schema: 'REFERENCED_SCHEMA'});
+
+  jest.mock('swagger-parser', () => ({
+    validate: mockValidate,
+    bundle: mockBundle
+  }));
+
+  const Swagger = require('../swagger');
+
+  it('should expose swagger object', () => {
+    expect(Swagger).toBeDefined();
+  });
+
+  describe('when swagger path is not passed', () => {
+    it('should throw error',async () => {
```

Comment:
@bhaskarmelkani Why this operation is async?

Reply:
Ah yeah, its not required. First was also testing validate with it so was required.
But then seperatad the tests.
Removed